<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SecurityLabClass</class>
 <widget class="QMainWindow" name="SecurityLabClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SecurityLab</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="AdminPage" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>301</height>
     </rect>
    </property>
    <widget class="QTableView" name="users_view">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>411</width>
       <height>301</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="adm_add_user_btn">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>10</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Add user</string>
     </property>
    </widget>
    <widget class="QPushButton" name="adm_block_btn">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>40</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Block/unlock user</string>
     </property>
    </widget>
    <widget class="QPushButton" name="adm_reset_verification_btn">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>70</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>reset verification</string>
     </property>
    </widget>
    <widget class="QPushButton" name="adm_info_btn">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>130</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>INFO</string>
     </property>
    </widget>
    <widget class="QPushButton" name="adm_logout_btn">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>160</y>
       <width>75</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Logout</string>
     </property>
    </widget>
    <widget class="QPushButton" name="adm_reset_password">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>100</y>
       <width>101</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>reset password</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="LoginPage" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <widget class="QPushButton" name="login_sign_in_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>201</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>sign in</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_username_edt">
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>10</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="login_password_edt">
     <property name="geometry">
      <rect>
       <x>86</x>
       <y>40</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>40</y>
       <width>49</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>16</x>
       <y>10</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="UserPage" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>221</width>
      <height>121</height>
     </rect>
    </property>
    <widget class="QPushButton" name="user_reset_password_btn_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Reset password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="user_logout_btn_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>91</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>logout</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SecurityLab.qrc"/>
 </resources>
 <connections/>
</ui>
